statement.md

```diff
@@ -0,0 +1,84 @@
+# JupyterLab Accessibility Statement
+
+## Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+
+## ⚠️ Draft Accessibility Statement for JupyterLab
+
+This is an accessibility statement from <span class="basic-information organization-name">Jupyter accessibility contributors</span>.
+
+### Measures to support accessibility
+
+Jupyter accessibility contributors take the following measures to ensure accessibility of JupyterLab:
+
+* Include accessibility as part of our mission statement.
+* Provide continual accessibility training for our community.
+* Assign clear accessibility goals and responsibilities.
+* Employ formal accessibility quality assurance methods.
+
+### Conformance status
+
+The <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines (WCAG)</a> defines requirements for designers and developers to improve accessibility for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA. JupyterLab is non conformant with WCAG 2.1 level AA. Non conformant means that the content does not conform the accessibility standard.
+
+### Feedback
+
+We welcome your feedback on the accessibility of JupyterLab. Please let us know if you encounter accessibility barriers on JupyterLab:
+
+* [Write an issue on jupyter/accessibility](https://github.com/jupyter/accessibility/issues/new)
+* [Write an issue on jupyterlab/jupyterlab](https://github.com/jupyterlab/jupyterlab/issues/new) and request it be labeled [tag:accessibility](https://github.com/jupyterlab/jupyterlab/labels/tag%3AAccessibility)
+
+### Compatibility with browsers and assistive technology
+
+JupyterLab is designed to be compatible with the following technologies:
+
+* Windows, macOS, iOS, Android; Firefox, Chrome, Safari, Chromium browsers (mobile and desktop).
```

Comment:
Oh wow, I've never opened JupyterLab on my smartphone. I assumed it wouldn't work...